shell/packages/sandstorm-ui-grainlist/grainlist-client.js

```diff
@@ -123,12 +128,14 @@ SandstormGrainListPage.bulkActionButtons = function (showTrash) {
         },
 
         onClicked: function (ownedGrainIds, sharedGrainIds) {
-          ownedGrainIds.forEach((grainId) => {
+          const grainIds = ownedGrainIds.concat(sharedGrainIds);
+
+          grainIds.forEach((grainId) => {
```

Comment:
Is the idea here that we'll call both `deleteGrain` and `forgetGrain` on each grain to delete, regardless of whether we own the grain or have ApiTokens for it?  I find it a bit weird that we're sending requests over the wire that we know a priori will fail here.